app/app.go

```diff
@@ -228,8 +229,9 @@ type App struct {
 	ScopedIBCKeeper      capabilitykeeper.ScopedKeeper
 	ScopedTransferKeeper capabilitykeeper.ScopedKeeper
 
-	BlobKeeper blobmodulekeeper.Keeper
-	QgbKeeper  qgbmodulekeeper.Keeper
+	BlobKeeper        blobmodulekeeper.Keeper
+	QgbKeeper         qgbmodulekeeper.Keeper
+	ParamFilterKeeper paramfilter.Keeper
```

Comment:
Not sure if this is a `Keeper` per say as it doesn't persist any data (i.e. nothing gets added to "state"). It feels more like some form of middleware (especially as it wraps the param keeper).

Reply:
good point! refactored in https://github.com/celestiaorg/celestia-app/pull/1694/commits/1a9506617fd5d49b6e7d26a5d8b990c420f0a72d